<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>915</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据制造器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pb_changesettingisshow">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>隐藏设置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>742</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_output">
           <property name="text">
            <string>输出</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_run">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>执行</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_setting" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <widget class="QCheckBox" name="cb_ssh">
         <property name="geometry">
          <rect>
           <x>1</x>
           <y>0</y>
           <width>90</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>使用ssh连接</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_dbtype">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>0</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>数据库</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MySQL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Oracle</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="le_ip">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>ip</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_port">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>数据库端口</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_dbname">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>数据库名</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_sshpassword">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>ssh密码</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_tablename">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>表名</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_sshusername">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>720</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>ssh用户名</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_localport">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>本地端口</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_sshport">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>ssh端口</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_dbpassword">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>数据库密码</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_dbusername">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>数据库用户名</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_alldatascount">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>数据总数</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_oneturndatascount">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>一轮数据条数</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_fields">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>711</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>字段名</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_choosesetting">
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>0</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <item>
          <property name="text">
           <string>选择设置</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pb_savesetting">
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>0</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>保存设置</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_deletesetting">
         <property name="geometry">
          <rect>
           <x>720</x>
           <y>0</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>删除设置</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="tab_importanddeclare">
         <attribute name="title">
          <string>导入/声明</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
        <widget class="QWidget" name="tab_fun">
         <attribute name="title">
          <string>函数主体</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
        <widget class="QWidget" name="tab_extraclass">
         <attribute name="title">
          <string>自定义类</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </widget>
        <widget class="QWidget" name="tab_setting">
         <attribute name="title">
          <string>设置</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_environment">
            <property name="text">
             <string>环境选择 (默认使用环境变量): </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_javac" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="le_javac">
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>javac.exe, 可拖拽至此</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_findjavac">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_savejavac">
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_java" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="le_java">
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>java.exe, 可拖拽至此</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_findjava">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_savejava">
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_findall" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pb_findall">
               <property name="text">
                <string>自动录入</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_viewclasses">
            <property name="text">
             <string>浏览自定义类: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_viewclassesandcontent" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QScrollArea" name="sa_viewclasses">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="widget_viewclasses">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>148</width>
                    <height>301</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>9</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>9</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="vl_viewclass">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="te_classcontent">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_deleteclass">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
